### ISSUE TITLE
metric with tags doesn't work

### ISSUE THREAD

**Issue by dumbdonkey** [graphite-project/carbon#832]
I use the graphite in Docker (this one:<https://github.com/graphite-project/docker-graphite-statsd>)

![image](https://user-images.githubusercontent.com/16414369/50293761-47afc400-04af-11e9-9cc0-7d5184769b85.png)

when I send a metric without tags, everything seemed good. well, if I add tags with metric I can't found my metrics anymore.

I have checked the carbon log but nothing found

Plz give me a hand, thanks a lot!

**Comment by deniszh** [comment]
Yes, that's how it works for now. Tagged metrics' name is hashed and resides under `_tagged` directory. So, dashboard doesn't currently support tagged metrics, you'll need e.g. Grafana to render it properly.

**Comment by deniszh** [comment]
See https://github.com/graphite-project/carbon/issues/813

**Comment by tmdoit** [comment]
Right, I lost two hours on trying to resolve this "problem".
